deis/manifests/deis-builder-rc.yaml

```diff
@@ -21,8 +21,6 @@ spec:
           ports:
             - containerPort: 2223
               name: ssh
-            - containerPort: 3000
-              name: fetcher
           env:
             - name: "EXTERNAL_PORT"
               value: "2223"
```

Comment:
@aledbf let's leave this as-is for now, since it uses the `quay.io/deis/builder:2.0.0-alpha` image, which still has the fetcher in it